### PR TITLE
Make v3 ownable

### PR DESCRIPTION
**Pull request by ryley-o**
## Change Summary
- Implements OpenZeppelin's `Ownable` of V3 core.
- Adds a public getter `admin()` for backwards-compatibility purposes only, which returns the contract owner.
- Adds changelog to track core changes from V1 -> V3 core

## Motivation
The main benefit here is better automatic integration with third-party OpenSea/aggregators.

### CODE REVIEW

**Comment by jakerockland** [comment]
#207 has been merged, making this diff'd off of main.

**Comment by jakerockland** [comment]
If it is ok @ryley-o, will let you resolve the merge conflicts here before I do my review?

**Comment by ryley-o** [comment]
@jakerockland ready for you 😄

**Review by jakerockland — APPROVED** [review]
LGTM! 🚢

File: contracts/GenArt721CoreV3.sol [hunk 1]
```diff
@@ -667,4 +652,21 @@ contract GenArt721CoreV3 is ERC721, IGenArt721CoreContractV3 {
...
     }
+
+    function owner()
+        public
+        view
+        override(Ownable, IGenArt721CoreContractV3)
+        returns (address)
+    {
+        return Ownable.owner();
+    }
```
Comment (jakerockland) [inline contracts/GenArt721CoreV3.sol:663]: Nit: can we add a documentation comment here that clarifies that by inheriting Ownable, the owner is set to the deployer by default on deployment and link to this: https://docs.openzeppelin.com/contracts/2.x/api/ownership#Ownable ? 

Alternatively may make more sense to add this note in the initializer; WDYT?

Comment (ryley-o) [reply contracts/GenArt721CoreV3.sol:669]: good catch - added docstrings to this function, noted deployment behavior, and linked to OpenZeppelin docs for v4 ownable!

Comment (jakerockland) [reply contracts/GenArt721CoreV3.sol:669]: TYVM!! :)

**Pull request merged**